apps/playground/src/app/components/forms/input-box/input-box.component.html

```diff
@@ -672,11 +671,18 @@
           formControlName="errorAutofillTextareaFormField"
           name="address"
           autocomplete="street-address"
+          required
         >
 Value</textarea
         >
       </sky-input-box>
     </div>
+
+    <sky-form-errors
```

Comment:
We talked offline about putting a `hintText` example in the `apps/code-examples` SPA to make sure the hint text and errors work well together.

Reply:
Also, might be good to create a visual test of both hint text and errors on the same input box to ensure the margins never change between PRs.

Reply:
Added the example.

That visual test already exists.